examples/2_Intermediate/B_internal_normal.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import os
+from simsopt.mhd.virtual_casing import VirtualCasing
+
+"""
+This example illustrates the virtual casing calculation, in which
+we compute the contribution to the magnetic field on the plasma
+boundary due to current inside the boundary, excluding currents
+outside the plasma. Typically this calculation is run once at the end
+of a stage-1 optimization (optimizing the plasma shape) before
+starting the stage-2 optimization (optimizing the coil shapes). It is
+only required for finite-beta plasmas, not for vacuum fields.
+
+This example requires that the python virtual_casing module is installed.
+"""
+
+filename = os.path.join(os.path.dirname(__file__), '..', '..',
+                        'tests', 'test_files', 'wout_li383_low_res_reference.nc')
+
+# Only the phi resolution needs to be specified. The theta resolution
+# is computed automatically to minimize anisotropy of the grid.
+vc = VirtualCasing.from_vmec(filename, nphi=180)
+
```

Comment:
This example would be confusing to the users because it appears that nothing is happening after instantiating the VirtualCasing class. My suggestion would to be print some of the important quantities and add comments stating how those quantities are used.

Reply:
A bit more is done now in this example.